conf/config.inc.php

```diff
@@ -150,7 +150,8 @@
 $mail_priority = 3;
 
 # Language
-$lang ="en";
+$lang = "en";
+$allowed_lang = "";
```

Comment:
Please initialize `$allowed_lang` as an array
```suggestion
$allowed_lang = array()
```

Reply:
Fixed!